transport-classes-io_uring/pom.xml

```diff
@@ -55,6 +58,63 @@
 
   <build>
     <plugins>
+      <plugin>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <version>3.13.0</version>
+        <!--
+          We need to compile everything except the IoUring class with Java9+
+          The reason for this is that we want still be able to use IoUring.ensureAvaibility() on Java8.
+        -->
+        <executions>
+          <execution>
+            <id>default-compile</id>
+            <goals>
+              <goal>compile</goal>
+            </goals>
+            <configuration>
+              <compilerVersion>1.8</compilerVersion>
```

Comment:
The Java 8 compiler can produce Java 9 byte code?

Reply:
Fixed